pkg/ctxutil/ctxutil_test.go

```diff
@@ -111,6 +111,22 @@ func TestCommitMessage(t *testing.T) {
 	assert.Equal(t, "", GetCommitMessage(WithCommitMessage(ctx, "")))
 }
 
+func TestCommitMessageBody(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+
+	ctx2 = AddToCommitMessageBody(AddToCommitMessageBody(WithCommitMessage(ctx, "foo"),"bar"),"baz")
```

Comment:
`ctx2` needs to be declared with `ctx2 := ...`.

Reply:
done!
(see comment later on)